cmake/build_mingw32.sh

```diff
@@ -8,7 +8,7 @@ else
 	CMAKE_OPTS="$CMAKE_OPTS -DLMMS_BUILD_MSYS=1"
 fi
 
-export PATH=$PATH:$MINGW/bin
+export PATH=$MINGW/bin:$PATH
```

Comment:
> `export PATH=$MINGW/bin:$PATH`
> > Will these changes affect linux builds though?

They should be benign and Travis-CI passes, which is a good sign.

What this change does is force the path to prefer mingw's BIN **first** to any system BIN.  I'd expect this behavior as well and we use it in other areas.  If I were to harbor a guess, I'd say this is just an oversight that's been copied and pasted since our first builds (see https://github.com/lmms/lmms/commit/d3516cd5c51d88b1ba6bd0428c45137552f98186#diff-7e1f009e6b046eedd463b471771e7a05R2)